test/api/database-issues.test.js

```diff
@@ -0,0 +1,36 @@
+const request = require("supertest");
+const { setupDatabaseIssueApp, getServer } = require("../../app-test");
+const isDocker = require("is-docker");
+
+let app = null;
+
+/**
+ * Remove and close the db and server.
+ */
+afterAll(async () => {
+  if (!!app) {
+    getServer().close();
+  }
+  jest.clearAllTimers();
+});
+
+async function getOrCreateDatabaseIssueApp() {
+  if (!app) {
+    app = await setupDatabaseIssueApp();
+  }
+  return app;
+}
+
+jest.mock("../../server_src/runners/githubClient");
+
+describe("DatabaseIssue server", () => {
+  it("should return database issue page when no database is connected", async () => {
+    app = await getOrCreateDatabaseIssueApp();
+
+    const res = await request(app).get("/").send();
+    expect(res.statusCode).toEqual(200);

```

Comment:
Test code and some content. Tbh this is where these kinds of tests start to fail a bit in their capacity.
We'd need to run the code in browser: karma with a headless chrome webdriver f.e.